evm/src/Transceiver/WormholeTransceiver/WormholeTransceiver.sol

```diff
@@ -250,4 +246,38 @@ contract WormholeTransceiver is
         checkFork(wormholeTransceiver_evmChainId);
         return getWormholePeer(vm.emitterChainId) == vm.emitterAddress;
     }
+
+    function _encodePayload(
+        address caller,
+        bytes32 recipientNttManagerAddress,
+        bytes memory nttManagerMessage,
+        bool isSpecialRelayer
+    ) internal view returns (
+        TransceiverStructs.TransceiverMessage memory,
+        bytes memory
+    ) {
+        // Transceiver payload is prefixed with 2 bytes
+        // representing the version of the payload.
+        // The rest of the bytes are the -actual- payload data.
+        // In payload v1, the payload data is a boolean
+        // representing whether the message should be picked
+        // up by the special relayer or not.
+        bytes memory transceiverPayload = abi.encodePacked(
+            uint16(1),
```

Comment:
Is setting this to 1 technically correct?

Reply:
I thought it'd be the `TranseiverInstruction` from the previous call? 

I don't think the Wormhole Transceiver actually processes this field atm though. It's more for the way in than the way out.

Reply:
It's only in the wormhole transceiver payload to make life for the specialized relayer easier: Namely, so it can look at the VAA alone to figure out if it is on the hook for relaying it or not. It has no relevance beyond that.

Reply:
If we use `TransceiverInstruction` we are leaving it up to the client to pass the right payload, this would be worst in terms of trust for the offchain relayer (because we'd now have to trust that the client offchain code, which we can't audit as easily as inspecting a verified contract)